assets/js/blocks/checkout/inner-blocks/checkout-pickup-options-block/block.tsx

```diff
@@ -118,7 +119,7 @@ const Block = (): JSX.Element | null => {
 
 	// Get pickup locations from the first shipping package.
 	const pickupLocations = ( shippingRates[ 0 ]?.shipping_rates || [] ).filter(
-		( { method_id: methodId } ) => methodId === 'pickup_location'
+		isPackageRateCollectable
```

Comment:
`isPackageRateCollectable` and `hasCollectableRate` seem to handle similar logic, why can't we use just one?